contracts/rng/RNGInterface.sol

```diff
@@ -1,10 +1,10 @@
 pragma solidity ^0.6.4;
 
 interface RNGInterface {
-  event RandomNumberRequested(uint256 indexed id, address indexed sender, address token, uint256 budget);
-  event RandomNumberCompleted(uint256 indexed id, bytes32 randomNumber);
+  event RandomNumberRequested(uint32 indexed id, address indexed sender, address token, uint256 budget);
+  event RandomNumberCompleted(uint32 indexed id, uint256 randomNumber);
 
-  function requestRandomNumber(address token, uint256 budget) external returns (uint256);
-  function isRequestComplete(uint256 id) external view returns (bool);
-  function randomNumber(uint256 id) external view returns (bytes32);
+  function requestRandomNumber(address token, uint256 budget) external returns (uint32, uint32);
```

Comment:
When it's more than one return value it's good to give them names

Reply:
Fixed in RNG package repo, which will be pulled into this repo with:
> POOL-246 - Publish an `npm` package for the RNG service